app/status/views.py

```diff
@@ -11,14 +12,16 @@ def status_no_db():
     try:
         return jsonify(
             status="ok",
-            version=utils.get_version_label(),
-            db_version=utils.get_db_version()
+            version=get_version_label(),
+            db_version=utils.get_db_version(),
```

Comment:
`get_version_label()` is used by all of the apps using exactly the same code. 
`get_db_version()` is only used by this one, so I figured it didn't make sense to bring it into utils.
Does it?